src/axe-tool-property-provider-21.ts

```diff
@@ -3,29 +3,29 @@
 import * as Sarif from 'sarif';
 import { WcagGuid } from './wcag-taxonomy-provider';
 
-export function getAxeToolProperties21(): Sarif.Run['tool'] {
+export function getAxeToolProperties21(): Sarif.ToolComponent {
     return {
-        driver: {
-            name: 'axe-core',
-            fullName: 'axe for Web v3.2.2',
-            shortDescription: {
-                text:
-                    'An open source accessibility rules library for automated testing.',
-            },
-            version: '3.2.2',
-            semanticVersion: '3.2.2',
-            informationUri: 'https://www.deque.com/axe/axe-for-web/',
-            downloadUri: 'https://www.npmjs.com/package/axe-core/v/3.2.2',
-            properties: {
-                'microsoft/qualityDomain': 'Accessibility',
-            },
-            supportedTaxonomies: [
-                {
-                    name: 'WCAG',
-                    index: 0,
-                    guid: WcagGuid,
-                },
-            ],
+        name: 'axe-core',
+        fullName: 'axe for Web v3.2.2',
+        shortDescription: {
+            text:
+                'An open source accessibility rules library for automated testing.',
         },
+        version: '3.2.2',
+        semanticVersion: '3.2.2',
+        informationUri: 'https://www.deque.com/axe/axe-for-web/',
+        downloadUri: 'https://www.npmjs.com/package/axe-core/v/3.2.2',
+        properties: {
+            'microsoft/qualityDomain': 'Accessibility',
+        },
+        supportedTaxonomies: [getAxeToolSupportedTaxonomy()],
+    };
+}
+
+export function getAxeToolSupportedTaxonomy(): Sarif.ToolComponentReference {
```

Comment:
This data is tightly coupled to the data in wcag-taxonomy-provider - maybe that would be a better home for it? (probably with a name more like `wcagTaxonomyReference`)